en-US/Design.xml

```diff
@@ -937,46 +937,63 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 		 <formalpara id="acronyms">
 			<title>Acronyms</title>
 			 <para>
-				What are acronyms anyway?
-        They are similar to abbreviations and initialisms but they are pronounced as a word.
-        An acronym is a word that is formed from the initial letters of a name, such as ROM for <emphasis>R</emphasis>ead-<emphasis>O</emphasis>nly <emphasis>M</emphasis>emory, or by combining initial letters or part of a series of words, such as LILO for <emphasis>LI</emphasis>nux <emphasis>LO</emphasis>ader.
-        <acronym>COBOL</acronym> is the acronym for Common Business-oriented Language, and <acronym>POP</acronym> is the acronym for Post Office Protocol.
+        		An acronym is a word that is formed from the initial letters of a name, such as ROM for <emphasis>R</emphasis>ead-<emphasis>O</emphasis>nly <emphasis>M</emphasis>emory, or by combining initial letters or part of a series of words, such as LILO for <emphasis>LI</emphasis>nux <emphasis>LO</emphasis>ader.
+        		<acronym>COBOL</acronym> is the acronym for Common Business-oriented Language, and <acronym>POP</acronym> is the acronym for Post Office Protocol.
+			</para>
+
+		</formalpara>
+		 <formalpara id="initialisms">
+			<title>Initialisms</title>
+			 <para>
+        		An initialism is an abbreviation that consists of the first letters of words in a phrase, syllables, or some combination thereof. Each character is pronounced separately. For example, FTP is an initialism for File Transfer Protocol.
 			</para>
 		</formalpara>
+		 <para>
+				Consider pronunciation when using articles. See <xref linkend="articles" /> for more information.
```

Comment:
It looks like pronunciation and article use is addressed in the following section. If so, consider removing this sentence.